node/dataSources/session.ts

```diff
@@ -40,7 +41,8 @@ export class SessionDataSource extends RESTDataSource<Context> {
     forEachObjIndexed(
       (value: string, header) => request.headers.set(header, value),
       {
-        ...segment && {Cookie: `vtex_segment=${segment}`},
+        'Content-Type': 'application/json',
+        'Cookie': headers.cookie,
```

Comment:
Need to make sure segment is included.

Reply:
It is already in the cookie on my tests.